stacks_queues/queue.py

```diff
@@ -15,48 +18,86 @@ def __init__(self):
         self.front = -1
         self.rear = -1
         self.size = 0
-      
 
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
+        if (self.front == 0 and self.rear == self.buffer_size - 1) or (self.rear == (self.front-1) % (self.buffer_size-1)):
```

Comment:
Works, but could be simplified a bit

```suggestion
        if ((self.rear + 1) % self.buffer_size == self.front):
```